src/policy/feerate.cpp

```diff
@@ -20,7 +19,6 @@ CFeeRate::CFeeRate(const CAmount& nFeePaid, size_t nBytes_)
 
 CAmount CFeeRate::GetFee(size_t nBytes_) const
 {
-    assert(nBytes_ <= uint64_t(std::numeric_limits<int64_t>::max()));
```

Comment:
How is this `assert` save to remove for 64-bit?

Reply:
Oh, my bad!

Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/19735#issuecomment-834304120).